webdriver-spec.html

```diff
@@ -2306,6 +2298,40 @@
  the operating system notaion of a “window”
  or the DOM <a><code>Window</code> object</a>.
 
+<p>The <a>top-level browsing context</a> is said
+ to be <dfn>no longer open</dfn> if it has been <a>discarded</a>.
+
+<p>Each <a>browsing context</a> has an associated
+ <dfn data-lt="window handle|window handles">window handle</dfn> (string)
+ uniquely identifying it.
+ It must not be "<code>current</code>".
+
+<p>The <dfn>web window identifier</dfn>
+ is the string constant "<code>window-fcc6-11e5-b4f8-330a88ab9d7f</code>".
+
+<p>The <dfn>web frame identifier</dfn>
```

Comment:
As I [explained in my email](https://lists.w3.org/Archives/Public/public-browser-tools-testing/2016AprJun/0005.html) it lets clients infer type information so they would know not to accept a frame as argument to *Switch To Window*.